### PR TITLE
Create news i18n

### PR DESCRIPTION
**Pull request by greatpanda1603**

### CODE REVIEW

File: src/app/component/eco-news/components/create-news/drag-and-drop/drag-and-drop.component.scss [hunk 1]
```diff
@@ -89,23 +92,23 @@ img {
...
-@media screen and (max-width: 880px) {
-    .dropzone {
-        width: 280px;
-    }
-}
+//@media screen and (max-width: 880px) {
+//    .dropzone {
+//        width: 280px;
+//    }
+//}
```
Comment (mplesha) [inline src/app/component/eco-news/components/create-news/drag-and-drop/drag-and-drop.component.scss:99]: Don't comment code

File: src/app/component/eco-news/components/create-news/drag-and-drop/drag-and-drop.component.scss [hunk 2]
```diff
@@ -89,23 +92,23 @@ img {
...
-@media screen and (max-width: 576px) {
-    .dropzone {
-        width: 330px;
-    }
-}
+//@media screen and (max-width: 576px) {
+//    .dropzone {
+//        width: 330px;
+//    }
+//}
```
Comment (mplesha) [inline src/app/component/eco-news/components/create-news/drag-and-drop/drag-and-drop.component.scss:111]: Don't comment code

File: src/app/component/eco-news/components/create-news/drag-and-drop/drag-and-drop.component.scss (hunk 1 above)
Comment (greatpanda1603) [reply src/app/component/eco-news/components/create-news/drag-and-drop/drag-and-drop.component.scss]: Commented code was removed

File: src/app/component/eco-news/components/create-news/drag-and-drop/drag-and-drop.component.scss (hunk 2 above)
Comment (greatpanda1603) [reply src/app/component/eco-news/components/create-news/drag-and-drop/drag-and-drop.component.scss]: Commented code was removed

File: src/app/component/eco-news/components/create-news/create-news.component.ts [hunk 3]
```diff
@@ -30,8 +30,9 @@ export class CreateNewsComponent implements OnInit {
     {name: 'Ads', isActive: false}
   ];
 
-  public activeLanguage = 'en';
-  private date: Date = new Date();
+  private year: number = new Date().getFullYear();
```
Comment (mplesha) [inline src/app/component/eco-news/components/create-news/create-news.component.ts:33]: Actually, you are using the variable <code>year</code> in the template, e.g. here:
```html
 <div class="date">
                <p><span>{{ 'create-news.date' | translate }}</span>
                  <span>{{day}} {{'date.months.' + [month] | translate }} {{year}} {{'date.year' | translate }}</span></p>
                <p><span>{{ 'create-news.author' | translate }}</span><span>{{ 'create-news.author-name' | translate }}</span></p>
            </div>
```
So, it shouldn't be <code>private</code>

File: src/app/component/eco-news/components/create-news/create-news.component.ts [hunk 4]
```diff
@@ -30,8 +30,9 @@ export class CreateNewsComponent implements OnInit {
...
+  private day: number = new Date().getDate();
+  private month: number = new Date().getMonth();
```
Comment (mplesha) [inline src/app/component/eco-news/components/create-news/create-news.component.ts:35]: Actually, you are using the variable <code>year</code> and other in the template, e.g. here:
```html
 <div class="date">
                <p><span>{{ 'create-news.date' | translate }}</span>
                  <span>{{day}} {{'date.months.' + [month] | translate }} {{year}} {{'date.year' | translate }}</span></p>
                <p><span>{{ 'create-news.author' | translate }}</span><span>{{ 'create-news.author-name' | translate }}</span></p>
            </div>
```
So, they shouldn't be <code>private</code>

**Comment by mplesha** [comment]
Resolve conflicts

**Pull request merged**